internal/handlers/telegram/handler_test.go

```diff
@@ -0,0 +1,40 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestDocument(t *testing.T) {
+	correct := "Someone shared a file (Random File) on Telegram with caption: 'Random Caption'."
+	updateObj := &tgbotapi.Update{
+		Message: &tgbotapi.Message{
+			From: &tgbotapi.User{
+				FirstName: "Someone",
+			},
+			Document: &tgbotapi.Document{
+				FileID: "Random File",
+			},
+			Caption: "Random Caption",
+		},
+	}
+	// Call the String() function which checks if a username(Optional) exists
+	updateObj.Message.From.UserName = updateObj.Message.From.String()
+
+	// Assign the MimeType(Optional) to FileID(Required) if it does not exists
+	if updateObj.Message.Document.MimeType == ""{
+		updateObj.Message.Document.MimeType = updateObj.Message.Document.FileID
```

Comment:
We don't want to set the mime type to anything if it doesn't exist. We just skip over it.

Reply:
I moved it to the `handler.go`.